ush/splitPlotYaml.py

```diff
@@ -29,6 +29,61 @@ def removeKey(d, keys):
     return r
 
 
+def check_plotlist(type, logger, plotlist, sat=None, instr=None, obstype=None):
+    """
+    Parse plotlist for missing required elements.
+
+    Parameters:
+        type (str): type of plot
+        logger (Logger):  logger for output warnings
+        plotlist (list):  list of requested plot specifications
+        sat (str): satellite name (deafault None)
+        instr (str); instrument name (default None)
+        obstype (str): observation type namea (default None)
+
+    Return:
+        True if no required elements are missing from plotlist, else False
+    """
+
+    sat_reqs = {'rad': ['plot', 'times', 'channels', 'component', 'run'],
+                'ozn': ['plot', 'times', 'levels', 'component', 'run']}
+    min_reqs = ['plot', 'times', 'run']
+    obs_reqs = ['plot', 'times', 'datatypes', 'run']
+
+    rtn_val = True
+    missing = []
+    match type:
+        case 'sat':
+            if 'rad' in plotlist.get('plot'):
+                reqs = sat_reqs.get('rad')
+            else:
+                reqs = sat_reqs.get('ozn')
+
+            for val in reqs:
+                if val not in plotlist:
+                    missing.append(val)
+                    plot_info = (f" {instr} {sat} {plotlist.get('plot')}")
+
+        case 'min':
+            for val in min_reqs:
+                if val not in plotlist:
+                    missing.append(val)
+                    plot_info = (f" {plotlist.get('plot')}")
+
+        case 'obs':
+            for val in obs_reqs:
+                if val not in plotlist:
+                    missing.append(val)
+                    plot_info = (f" {obstype}, {plotlist.get('plot')}")
+
+    if missing:
```

Comment:
`missing` is the list of items missing from the `plotlist` and it's initialized to an empty list.  The return value is set to a default of True and then set to False if `missing` is not empty.  I can change this to` if length(missing) > 0`  instead of `if missing`.  That would clearly indicate that `missing` isn't a boolean value, something python seems adept at obfuscating.

Reply:
yeah I think that would be preferred